infrastructure/src/test/java/org/corfudb/infrastructure/management/ClusterStateTest.java

```diff
@@ -20,29 +20,6 @@
 
 public class ClusterStateTest {
 
-    @Test
```

Comment:
why have you deleted this test?

Reply:
Cause it assumes that cluster state that has multiple epoch is invalid and now its valid after this change.